File: legds/fun1-2.1/c/main.c
#include "main.h"

int main() {
	int length;

	printf("Please enter length of array\n");
	scanf("%d",&length);
	List *list = MakeList(length);

	printf("Please enter you want enter's number of element\n");
	int last;
	scanf("%d",&last);

	printf("Please enter value of array\n");
	for(int i = 0;i < last;i ++) {
		scanf("%d",&list->arr[i]);
		list->last ++;	
	}

	int insert_value;
	printf("Please enter value of insert\n");
	scanf("%d",&insert_value);

	int re = Insert(list,insert_value);
	if(re > 0) {
		printf("After insertion\n");
		for(int i = 0;i <= list->last;i ++) {
			int c = (i == (list->last))?('\n'):(' ');
			printf("%d%c",list->arr[i],c);
		}
	}

	return 0;
}

File: legds/fun1-2.1/java/Main.java
import java.util.ArrayList;
import java.util.Scanner;
import src.Insert;

public class Main {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		ArrayList<Integer> arr = new ArrayList<>();
		
		System.out.printf("Please enter number of element%n");
		int number = scan.nextInt();

		System.out.printf("Please enter element of array%n");
		for(int i = 0;i < number;i ++) {
			int element = scan.nextInt();
			arr.add(element);
		}

		System.out.printf("Pleade enter element of insertion%n");
		int insert = scan.nextInt();

		Insert.insert(arr, insert);

		System.out.printf("After Insert%n");
		for(int i = 0;i < arr.size();i ++) {
			char c = (i == (arr.size() - 1))?('\n'):(' ');
			System.out.printf("%d%c",arr.get(i),c);
		}

		scan.close();

	}
}
